### ISSUE TITLE
Teleport doesnt teleport to Port Osmos (Bug)

### ISSUE THREAD

**Issue by Thafoxes** [Akebi-Group/Akebi-GC#529]
Was using pr/469.
Video proof below, 

https://youtu.be/i5g83wGc1YU

**Comment by harlanx** [comment]
Best to use on map sections with unlocked waypoints https://github.com/Akebi-Group/Akebi-GC/issues/526#issuecomment-1228450942

**Comment by thelostabbey** [comment]
yep, what the others said, you need a waypoint unlocked and within a minimum distance.  the hack replaces the coordinates of any nearby waypoint with the location you choose.  if when you use map teleport and it returns to game it means it couldn't find one on its own. if so, go back to the map and teleport to any unlocked waypoint you have (I recommend the closest one to your original choice) and it'll use that one to now teleport you to the place you chose.

**Comment by Taiga74164** [comment]
cant reproduce bro
https://streamable.com/bem393

**Comment by Thafoxes** [comment]
> cant reproduce bro https://streamable.com/bem393

Best place to reproduce it here:
https://streamable.com/graj9l video for more info on the bug. As thelostabbey said, yeah I think that caused the problem
